chat-plugins/poll.js

```diff
@@ -312,6 +312,7 @@ exports.commands = {
 	pollhelp: ["/poll allows rooms to run their own polls. These polls are limited to one poll at a time per room.",
 				"Accepts the following commands:",
 				"/poll create [question], [option1], [option2], [...] - Creates a poll. Requires: % @ * # & ~",
+				"/poll htmlcreate [question], [option1], [option2], [...] - Creates a poll with HTML. Each option can be on a separate line when the command is being sent. Requires: # & ~",
```

Comment:
"Creates a poll with HTML." isn't exactly good English grammar here.

"Creates a poll, with HTML allowed in the question and options." is probably better.